### DISCUSSION TITLE
Documentation issues

### DISCUSSION THREAD

**Discussion by HeartlandTechie** [filamentphp/filament#1716] [General]
Can we clean up the documentation to have one complete coherent example?   The snippets without the blade template on the Forms side is frustrating - there are typos and I believe the example for creating a record references a create method  - where or how would that be a viable option?   Livewire normally suggests a submit function to get data . . . am I missing something?

**Comment by danharrin** [comment]
I've tried to vary the examples between sections to make them more contextual, what specially is wrong with what we have at the moment?

**Reply by HeartlandTechie** [reply]
Sometimes things get confusing . . . like an example (not necessarily of your software, but in general) . . . Blog -> Post -> Author etc...  but when you are looking at docs $this->post . . . is that the actual post model or is that a magic thing that has the post variables from the http request?  ARGH!

**Reply by danharrin** [reply]
Would adding missing types to those properties help so you can see that they are models and not anything else? I thought we added all of the model-based ones.

**Reply by HeartlandTechie** [reply]
A complete example with the snippets would be very nice . . . literally I have tried copying/pasting code and it doesn't run.  In one spot you have a "create" function . . . but not a "submit" function . . . so I have no idea how the blade template/livewire piece is working.
